### ISSUE TITLE
Update theme with new critical status color for light and dark modes

### ISSUE THREAD

**Issue by gregfuruiye** [hpe-design/design-system#989]
New colors for critical status to meet WCAG 2.1 AA compliance for both white and black text:

Light mode critical status color = FC5A5A 
Dark mode critical status color = D04F4E
